src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/GrunnlagsdataService.kt

```diff
@@ -36,12 +36,13 @@ class GrunnlagsdataService(private val grunnlagsdataRepository: GrunnlagsdataRep
         val behandling = behandlingService.hentBehandling(behandlingId)
         feilHvis(behandling.status.behandlingErLåstForVidereRedigering(),
                  HttpStatus.BAD_REQUEST) { "Kan ikke laste inn nye grunnlagsdata for behandling med status ${behandling.status}" }
-        slettGrunnlagsdataHvisFinnes(behandlingId)
+        privTransactional(behandlingId)
```

Comment:
Liker ikke dette metodenavnet noe særlig...

Reply:
Haha nei, den skulle absolut ikke med. Var for å teste `Detekt` 🐐